swiftwinrt/code_writers.h

```diff
@@ -48,7 +48,7 @@ namespace swiftwinrt
         }
         w.write("}\n");
 
-        w.write("extension %: ^@retroactive Hashable, ^@retroactive Codable {}\n\n", get_full_swift_type_name(w, type));
+        w.write("extension %: ^@retroactive Hashable, ^@retroactive Codable, ^@retroactive Sendable {}\n\n", get_full_swift_type_name(w, type));
```

Comment:
@tristanlabelle  i.e. I think this needs to be

```suggestion
        w.write("extension %: ^@retroactive Hashable, ^@retroactive Codable, ^@retroactive @unchecked Sendable {}\n\n", get_full_swift_type_name(w, type));
Comment
```

Reply:
Yes probably, will update. It's a bummer that Swift is inconsistent.

Reply:
🤔  it doesn't seem inconsistent to me, I think the error message is just poorly worded.

Reply:
Hmm I think I get it but yeah the message is misleading because `@retroactive` allows the conformance to be in a different module, which seems like a superset of `@unchecked` which is described as allowing the conformance to be in a different file.

```
conformance to 'Sendable' must occur in the same source file as struct '__x_ABI_CArcCoreWinRT_CAdhocHistogramType'; use '@unchecked Sendable' for retroactive conformance; this is an error in the Swift 6 language mode",
```